applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian_quadrilateral.hpp

```diff
@@ -610,6 +630,31 @@ class UpdatedLagrangianQuadrilateral
      */
     bool mFinalizedStep;
 
+    /**
+     * Container to store shape functions over whole timestep
+     */
+    Vector mN;
+
+    /**
+     * Container to store shape function gradients over whole timestep
+     */
+    Matrix mDN_DX;
+
+    /**
+     * Boolean for explicit time integration
+     */
+    bool mIsExplicit = false;
+
+    /**
+     * Boolean for stress update option
+     */
+    bool mIsUSFStressUpdate = false;
+
+    /**
+     * Boolean for central difference explicit time integration
```

Comment:
if it's just one line of comment you can also do /// to avoid two extra lines. Up to you.